docs/howto/CreateNewTerm.md

```diff
@@ -0,0 +1,48 @@
+# Creating a New Ontology Term in Protege
```

Comment:
https://oboacademy.github.io/obook/lesson/contributing_to_obo_ontologies/

I am not too worried about redundancy, as long as it is non-conflicting. The easiest way to achieve this is to have @nicolevasilevsky review all protege related howto guides.

Reply:
I'll make a list for @nicolevasilevsky to review